src/main/java/com/mycompany/app/application/StockQuoteApplication.java

```diff
@@ -0,0 +1,47 @@
+package com.mycompany.app.application;
+
+import java.util.*;
+import java.text.SimpleDateFormat;
+import com.mycompany.app.service.*;
+import com.mycompany.app.model.*;
+
+public class StockQuoteApplication {
+
+    /**
+     * A main method which enables program execution
+     *
+     * @param args an array that should contain as elements:
+     *       {@code String} representations of a symbol, start date, and end date
+     */
+    public static void main(String[] args) {
+
+        StockServiceFactory stockServiceFactory = new StockServiceFactory();
+        StockService basicService = stockServiceFactory.createStockService("BASIC");
+		
+        StockQuote stockQuote1 = basicService.getQuote(args[0]);
+		
+		Calendar startDate = formatDate(args[1]);
+		Calendar endDate = formatDate(args[2]);
+	
+        List<StockQuote> stockQuote2 = basicService.getQuote(args[0], startDate, endDate);
+	}
+  
+	 /**
+     * Format date string into a Calendar object
+     *
+     * @param string date to be formatted
+	 * @return a Calendar object
+     */
+	private static Calendar formatDate(String stringDate) {
+		
+		SimpleDateFormat sdf = new SimpleDateFormat("MM/dd/yyyy");
+		Calendar dt = Calendar.getInstance();
+		try {
+			dt.setTime(sdf.parse(stringDate));
+		}
+           catch (Exception e) {
+               e.printStackTrace();
```

Comment:
Yikes - this is not good. This is called dropping the exception. Your code should always handle the exception in some useful way. When you drop the exception (basically your are ignoring the error) you are just masking a problem that will still cause problems. Please review the class material on exception handling and let me know if you have any questions

Reply:
I have read the material on exception handling and I'm still uncertain how to proceed with coding to handle the exception.  Should I be catching a more specific exception like ParseException and printing a more specific message?  It seems like the Calendar object would be defaulted to the current date.